db/migrate/20210119122653_add_flat_view.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require './lib/views_schema'
+
+# Flattened view of the events database with one row per subject.
+# Metadata is excluded
+class AddFlatView < ActiveRecord::Migration[5.2]
+  def up
+    ViewsSchema.create_view(
+      'flat_events_view',
+      <<~SQL
+        SELECT
+          events.id AS wh_event_id,
+          events.uuid AS event_uuid_bin,
+          INSERT(INSERT(INSERT(INSERT(LOWER(HEX(events.uuid)),9,0,'-'),14,0,'-'),19,0,'-'),24,0,'-') AS event_uuid,
+          event_types.key AS event_type,
+          events.occured_at AS occured_at,
```

Comment:
Spelling typo that crept in and almost killed when debugging the UKB event warehouse!

Reply:
Ugh, tell me about it. I wasn't quite sure what to do here, and figured consistency with the existing tables is probably best. Hmm, suppose I could have both the correct and incorrect spellings.

Reply:
Yeah consistency is key, we can tackle that type another time.